Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs

```diff
@@ -8,7 +8,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.RiggedHandVisualizer
+namespace Microsoft.MixedReality.Toolkit.Input
```

Comment:
It seems like most of the other scripts in the SDK/Features/UX folder are in the `.UI` namespace. Should this be in that one too, or moved to a different folder?

Reply:
Our BaseHandVisualizer is in the .Input namespace, so I thought it would make sense to have this be in there too.